frontend/src/components/place/PlaceCheckinModal.vue

```diff
@@ -11,12 +11,14 @@
                         :key="index + 1"
                         :value="index + 1"
                         :icon="icon"
+                        :selected="isSelected(index + 1)"
                         @onHover="onHover"
+                        @onOut="onOut"
                         @onSelect="onSelect"
                     />
                 </div>
 
-                <span class="rating">{{ userRating }}/10</span>
+                <span class="rating">{{ userRating || place.myRating || 0 }}/10</span>
```

Comment:
not sure if we need myRating here

Reply:
we need it to highlight previously selected rating